channel-config/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,7 @@
+[[extensions]]
+type = "channel_config"
+handle = "{{ handle }}"
+{% if uid %}uid = "{{ uid }}"{% endif %}
+
+[[extensions.channel_requirements]]
+handle = "Shop"
```

Comment:
`handle` is kind of a loaded term especially with extension handles and app handles - should this just be `channel` to be really clear?

Reply:
I renamed it to `channel_handle`. This will ultimately end up in the existing `Channel#handle` field, so I'd like to keep `handle` in there if possible.